### ISSUE TITLE
Todesanimation Spieler

### ISSUE THREAD

**Issue by mk1x86** [Acknex/LottiVersusTheWorld#12]
Ich weiß nicht, ob es eine gibt, aber wir sollten eine haben. Kann ja theoretisch auch explodieren oder so, aber nur game over einblenden ist was wenig, denke ich.

Werde auch noch einen sterbesound machen

**Comment by simonschreibt-code** [comment]
wäre lustig wenn er in seine einzelnen polygone/dreiecke zerspringen könnte. ansonsten könnte es auch eine partikel-nova tun, falls das geht :)

**Comment by Slin** [comment]
Dann brauchen wir eine zweite (oder gleiche weil wir eh kein shading haben) Version des Players mit harten Edges, die wir dann entlang der normalen im shader bewegen könnten. Macht die Flugbahn etwas langweilig, könnte aber trotzdem ganz gut aussehen wenns schnell genug ist. Könnte man natürlich auch per animation lösen, aber ich vermute mal das will keiner machen :D.

**Comment by ikskuh** [comment]
Slin: Ich hab an das selbe gedacht, nur: Man kann ja Fake Gravity mit ner Flugbahn machen, eventuell sogar nen Bounce auf dem Boden mit den Polygonen. Aber ja, dafür brauchen wir das Modell in Edge-Split

**Comment by xwcg** [comment]
Könnte man unter umständen einfach die Polygone random ausblenden? Zerfallen ist zwar nett aber das ding nochmal in hard edges zu machen ist wahrscheinlich zu viel aufwand

**Comment by ikskuh** [comment]
Ne, leidet an der selben Krankheit. Wenn dann könnte man anfangen, einzelne Vertices auszublenden...

**Comment by xwcg** [comment]
das hoffentlich auch dazu führt dass die damit assoziierten polygone verschwinden? ^^

**Comment by ikskuh** [comment]
Ja, könnte passieren. Sieht dann halt weird aus, wenn das ganze nicht irgendwie gleichmäßig passiert, Kann halt sein, dass gleich ganz große flächen verschwinden oder nur ein kleiner kreis

**Comment by xwcg** [comment]
wenns schnell genug passiert fällt das niemandem auf, länger als 1 - 2 sekunden würde ich das auch nicht machen lassen.

**Comment by simonschreibt-code** [comment]
Im Notfall nehmen wir einfach einen Vorhandenen Explosions-Partikel-Effekt, färben die Partikel grün und machen den Effekt evtl noch 150% größer. Das könnte zumindest schonmal eine Minimal-Implementation sein.

**Comment by firoball** [comment]
Kann man das freundchen nicht vom Brett wichsen, beim Aufprall auf den Boden schnell removen und direkt ein paar tausend Dreiecke mit random scale Winkel drall und Richtung spawnen?

**Comment by mk1x86** [comment]
Keine Zeit mehr, close

**Comment by xwcg** [comment]
Können wir bitte nicht issues closen weil sie nicht in den 1.0 release kommen, dafür gibts labels

**Comment by mk1x86** [comment]
Wirst du denn einen 1.1 release machen?

**Comment by xwcg** [comment]
vielleicht, vielleicht nicht, aber das ist falsches handling von issues. Schlechte angewohnheiten und so. Erst gar nicht damit anfangen!

**Comment by firoball** [comment]
Ich mache heut Abend einen Release candidate 1.0.Den dann bitte erst testen, bevor wir ihn auf die Menschheit loslassen. Ich melde mich dann.Am 15.08.2017 9:48 vorm. schrieb mk1x86 <notifications@github.com>:Wirst du denn einen 1.1 release machen?
